app/javascript/app/pages/mitigation/mitigation-effects/summary/summary-component.jsx

```diff
@@ -118,11 +118,18 @@ class Summary extends PureComponent {
                   summarySelected && (
                   <div>
                     <p className={styles.label}>Action</p>
-                    <h2 className={styles.action}>
+                    <a 
+                      className={styles.action}
+                      href={`/mitigation?search=${summarySelected.action}`}
+                      target="_blank"
```

Comment:
Because the link is to a page inside the same website, I think it shouldn't open in a new tab.